### ISSUE TITLE
nysiis() struggles with non-ASCII characters that soundex() and metaphone() can handle

### ISSUE THREAD

**Issue by nchammas** [jamesturk/jellyfish#56]
Here's a simple repro with the non-ASCII character "ç":

```
>>> jellyfish.nysiis('ç')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: character U+ffffffff is not in range [U+0000; U+10ffff]
>>> jellyfish.soundex('ç')
'C000'
>>> jellyfish.metaphone('ç')
'K'
```

Is this expected?

I'm on Jellyfish 0.5.6 / Python 3.5.2.

**Comment by jamesturk** [comment]
Thanks for reporting, looking into this now, it seems to be an issue w/ the C implementation only.  If you need the Python implementation in the meantime you can import it directly via jellyfish._jellyfish.nysiis

**Comment by jamesturk** [comment]
fixed via #177
